.github/ISSUE_TEMPLATE.md

```diff
@@ -1,5 +1,33 @@
-Please include the following information in your ticket.
+Please [read these guidelines](http://ibm.biz/cdt-issue-guide) before opening an issue.
 
-- Cloudant (python-cloudant) version(s) that are affected by this issue.
-- Python version 
-- A small code sample that demonstrates the issue.
+<!-- Issues will be CLOSED IMMEDIATELY if the following template is not completed. -->
+
+## Bug Description
+
+### 1. Steps to reproduce and a small code sample that demonstrates the issue
+<!--
+EXAMPLE:
+1. Initialize client
+2. List DBs
+
+cc = new CloudantClient(...)
```

Comment:
This need to be updated to something like:
```
client = Cloudant(...)
dbs = client.all_dbs()
```

Reply:
Well the idea was that this example was the same pseudo-code in every repo so we didn't have to update it all the time. If you think that is too confusing we can do a similar thing to the snippets for versions and update it to something in keeping with the code in each repo.

I think the idea though is to show people what kind of code sample to provide, not how to use the specific lib.

Reply:
@ricellis Yea makes sense, let's keep it as pseudo-code.  Disregard the similar comments in the PRs of the other libraries.